webpack/webpack.config.base.js

```diff
@@ -208,6 +204,12 @@ const config = {
     new CopyWebpackPlugin({
       patterns: [{ from: 'node_modules/webextension-polyfill/dist/browser-polyfill.js' }],
     }),
+
+    new Dotenv({
+      allowEmptyValues: true,
+      systemvars: IS_PROD,
+    }),
+
     new webpack.DefinePlugin({
```

Comment:
We could likely replace `DefinePlugin` here and just use shared variables from `environment.ts`, but we seem to need `VERSION` in the webpack config to generate the correct manifest so that would create duplicate code.

Reply:
Let's run through these changes during our call later

Reply:
Yep, for sure, I was referencing suggestions similar to this which seems to be a common pattern: https://prateeksurana.me/blog/using-environment-variables-with-webpack/

Reply:
helpful link `systemvars` is the setting we want